db/seeds/0001_snp_plate_purposes.rb

```diff
@@ -72,9 +72,15 @@
 
 ActiveRecord::Base.transaction do
   # A couple of legacy pulldown types
+  # rubocop:todo Layout/LineLength
   PlatePurpose.create!(name: 'SEQCAP WG', cherrypickable_target: false) # Superceded by Pulldown WGS below (here for transition period)
+
+  # rubocop:enable Layout/LineLength
+  # rubocop:todo Layout/LineLength
```

Comment:
Should be consolidated

Reply:
Done, along with any others that needed consolidating.